README.md

```diff
@@ -7,7 +7,7 @@
 - Tested up to Wordpress: 6.6.1
 - Tested up to Woocommerce: 9.2.3
```

Comment:
Our compatibility version isn't up to date ?
- Wordpress : 6.7.1
- WooCommerce 9.6.2